schema/ast.go

```diff
@@ -164,6 +164,11 @@ type Sequence struct {
 	OwnedBy     string
 }
 
+type DefaultDefinition struct {
+	name  string
+	value *Value
```

Comment:
How about calling it `constraintName` for the best clarity? Also, I think it's better to clarify it's only for mssql, and should be put lower than `value` because `value` is the most main field here.

```suggestion
	value *Value
	constraintName string // only for MSSQL
```

Reply:
I agree your suggestion.
I changed the naming of the `sqlparser#ast#DefaultDefinition` in the same way.